testing/kyotocabinet/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Andrey L <innerspacepilot@gmx.com>
+# Maintainer:  Andrey L <innerspacepilot@gmx.com>
+pkgname=kyotocabinet
+pkgver=1.2.77
+pkgrel=0
+pkgdesc="Kyoto Cabinet is a library of routines for managing a database."
+url="https://fallabs.com/kyotocabinet/"
+arch="all"
+license="GPL-3.0-only"
```

Comment:
Looking at Void Linux's template, it is licensed under LGPL-2.1-or-later

Reply:
file COPYING inside source tree contains
GNU GENERAL PUBLIC LICENSE
Version 3, 29 June 2007

and README
================================================================
 Kyoto Cabinet: a straightforward implementation of DBM
 Copyright (C) 2009-2011 FAL Labs
================================================================


Please read the following documents with a WWW browser.
How to install Kyoto Cabinet is explained in the specification.

  README         - this file
  COPYING        - license (GPLv3)
  FOSSEXCEPTION  - license exception for FOSS
  LINKEXCEPTION  - license exception for specific libraries
  ChangeLog      - history of enhancement
  doc/index.html - index of documents

Reply:
You have to look in the source code headers to see if they are -only or -or-later. Note the section for -or-later:

``` 
the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or any later version.
```

in kcfile.cc (and all other files).

if it is -only then it will have `only` instead of `, or any later version`

Reply:
Ok.
I've set it to GPL-3.0-or-later